src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java

```diff
@@ -222,7 +222,7 @@ public void addNodeClinicalData(boolean header, boolean footer) {
                     }
 
                     if (BooleanUtils.isTrue(se.getLocked()))
-                        xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml("locked"));
+                        xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml("Locked"));
                     else if (BooleanUtils.isTrue(se.getSigned()))
                         xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml("signed"));
```

Comment:
is Signed supposed to start with uppercase letter too?

Reply:
It was not reported as an issue the signed one.

Reply:
Jigyasa says only Locked ones and not the signed ones.